script/src/syscalls/mod.rs

```diff
@@ -29,6 +29,7 @@ enum CellField {
     Capacity = 0,
     Data = 1,
     DataHash = 2,
+    Lock = 6,
```

Comment:
Move this item after `TypeHash = 5` for consistency.

Reply:
There are 2 ways to understand consistency here: first, we can sort them by the value of the enums, of course. But there's also another way which we can group the values based on the meeting, so we can do Data, DataHash, Lock, LockHash, this way it's easier to see what options are available here. Thoughts?